pkgdb2client/admin.py

```diff
@@ -437,12 +465,18 @@ def do_process(args):
                 action['id'], action['status']))
             return
 
+        package = None
         if action['action'] == 'request.package':
             try:
-                PKGDBCLIENT.get_package(
-                    action['info']['pkg_name'],
+                package_name = action['info']['pkg_name']
+                package = PKGDBCLIENT.get_package(
+                    package_name, branches=["master"]
+                )["packages"][0]
+
+                package = PKGDBCLIENT.get_package(
+                    package_name, branches=["master"],
                     namespace=action['info'].get('pkg_namespace', 'rpms'),
-                )
+                )["packages"][0]
```

Comment:
I'm seeing two queries to pkgdb saved in the same variable here, left over code?